### ISSUE TITLE
Questions about training methods

### ISSUE THREAD

**Issue by SatireY** [peract/peract#37]
Do you consider the different variants in your training? Does the training set include all the variants? Or did you save some of it for the test set? And the scene coordinate range question, also hope you can answer.

**Comment by MohitShridhar** [comment]
@SatireY, in PerAct experiments, `data_generator.py` uses `--all_variations` to generate datasets with randomly sampled variations. So a dataset of demos contains a mix of all RLBench variations of that task.
